ipu-plugin/pkg/ipuplugin/bridgeport.go

```diff
@@ -28,65 +28,82 @@ import (
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
-const (
-	outerVlanId = 0 // hardcoded s-tag
-)
-
+/*
+interfaces slice will be populated with PortIds on ACC(port representators->PRs on ACC, for Host VFs),
+for example, if ACC interface name is enp0s1f0d4, PortId(vportId) will be 4.
+Will also include port representators->PRs needed for Network Functions.
+*/
+var interfaces []uint
+var intfMap map[uint]bool
```

Comment:
What is the intfmap for? Comment describes interfaces slice but not the map I think

Reply:
Will ADD-> intfMap is a map, that has key:value, like between interfaceId and whether it is available(true or false) for use.

Reply:
Added Comment.